libraries/AP_ICEngine/AP_ICEngine.cpp

```diff
@@ -343,6 +376,25 @@ bool AP_ICEngine::throttle_override(uint8_t &percentage)
         percentage = (uint8_t)start_percent.get();
         return true;
     }
+
+    if (state == ICE_RUNNING && redline_flag && options & !(uint16_t(Options::DISABLE_REDLINE_LIMIT))) {
```

Comment:
```suggestion
    if (redline_flag && options & !(uint16_t(Options::DISABLE_REDLINE_LIMIT))) {
```
Could this be removed? It would be nice for this feature to work without the need for a starter to be enabled. The other way would be setting up a way to set the running state when the starter channel is none.